README.md

```diff
@@ -1,5 +1,81 @@
 Chitter Challenge
 =================
+A small web app mimicking Twitter that allows users to post to a public stream. Completed for the purpose of practising databases & web apps.
+
+## Setup
+Run
+```shell
+bundle install
+```
+to install gems
+
+Create databases and tables via psql using sql instructions in db/migrations. Remember to create the tables in both databases.
+
+## Database Connection
+I have setup the connection to the database with ActiveRecord. This should work once you have bundle installed.
+
+## Usage
+Run
+```shell
+rackup
+```
+and visit localhost:9292 to view online website. Feel free to sign up to post peeps and explore! You will need to sign 2 people up if you would like to tag them in your peep.
+
+Run
+```shell
+rspec
+```
+for testing.
+
+## Diagrams
+User Table
+|id|name|email|username|password|
+|--|--|--|--|-|
+
+Peep Table
+|id|message|created_at|user_id|
+|-|-|-|-|
+
+Tag Table
+|id|peep_id|user_id|
+|-|-|-|
+
+I wanted to call the user_id column in the tag table "tagged_user_id", and call "peep.tagged_users" to retrieve them, but couldn't quite figure out how to change the names via activerecord.
+
+## My learning
+
+#### Advice needed:
+Project Setup - I tried but haven't yet worked out how to run the database migrations using rake. Help needed here!
+
+MVC - I think some of my controller is too 'fat' but I'm not sure how to make them thinner. Not sure I've got the MVC balance correct as the views also seem to be doing quite a lot. Is flash[:notice] done correctly? I haven't used next or now. Also, should I have more views? For example, one view for viewing the peeps before logging in and another view for viewing the peeps once logged in? This would minimise some of the <% if %> statements in the views, but just not sure if it's considered to be a better approach or not. Any advice would be really appreciated.
+
```

Comment:
Are the flash messages working as you would like them to?

What is the difference between using the `.now` with flash messages and not? The code comments in https://github.com/SFEley/sinatra-flash#setting-up might be useful

Reply:
Yes I will explore this more, thank you